### ISSUE TITLE
LH should throw an error for non-function measures

### ISSUE THREAD

**Issue by gridaphobe** [ucsd-progsys/liquidhaskell#816]
In http://stackoverflow.com/questions/38938431/using-liquid-haskell-to-check-for-valid-tokens/38943118 the Q uses a "measure"

```
{-@ measure currTime :: UTCTime @-}
currTime :: UTCTime
currTime = UTCTime (ModifiedJulianDay 57614) 83924.978297
```

which is not a function. What does this mean to LH? I have no idea (possibly nothing?), but it's certainly not what you would expect (which IMO would be to create a logical value equal to currTime).

This should be a friendly error instead :)

**Comment by ranjitjhala** [comment]
Indeed, what _does_ happen? Am surprised we don't get an error message?
(Does it just crash with some panic?)

**Comment by nikivazou** [comment]
Surprised too it doesn't crash. Did you solve the stack overflow issue
though? O am traveling so kind of difficult to debug soon...
